AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -213,12 +208,23 @@ define([
                 return;
             }
             this.modelFilterValue(record.id);
+            this.applyFilter(record.id.toString());
+        },
+
+        /**
+         * Apply series or model id filter and scroll to top of the page
+         *
+         * @param {string} recordId
+         */
+        applyFilter: function (recordId) {
```

Comment:
Two different filters (`serie_id` and `model_id`) were replaced with single `model_id`